pkger/clone_resource.go

```diff
@@ -71,13 +71,13 @@ type cellView struct {
 	v influxdb.View
 }
 
-func convertCellView(cv cellView) chart {
+func convertCellView(cell influxdb.Cell) chart {
 	ch := chart{
-		Name:   cv.v.Name,
-		Height: int(cv.c.H),
-		Width:  int(cv.c.W),
-		XPos:   int(cv.c.X),
-		YPos:   int(cv.c.Y),
+		Name:   cell.View.Name,
```

Comment:
we'll want to `if cell.View != nil` yada yada before accessing the view stuffs. I missed this in the original convert func 🤦‍♂

Reply:
yeah, just caught this on my end as well haha